docs/tutorials/blog.md

```diff
@@ -896,28 +845,24 @@ TypeScript is mad again, let's add some types.
 
 ```tsx filename=app/models/post.server.ts lines=[2,8]
 // ...
-import type { Post } from "@prisma/client";
-export type { Post };
+import type { Prisma } from "@prisma/client"
 
 // ...
 
-export async function createPost(
-  post: Pick<Post, "slug" | "title" | "markdown">
-) {
+export async function createPost(post: Prisma.PostUncheckedCreateInput) {
```

Comment:
Prisma generates a type of input, I thought its more flexible to use than a `Pick`